CONTRIBUTING.md

```diff
@@ -17,6 +25,8 @@ This is a rough outline of what a contributor's workflow looks like:
 - Create a topic branch from where to base the contribution. This is usually master.
 - Make commits of logical units.
 - Make sure commit messages are in the proper format (see below).
+- Check your work after running all Unit and Regression Tests. You should run all the unit tests by hitting the following command
+    mvn clean install 
```

Comment:
I would mark the `mvn clean install` as TBD since we really don't know if that's what the commands will be yet, since there's no code here.

Reply:
Agree. I will modify it.